quipucords/fingerprinter/jboss_eap.py

```diff
@@ -218,8 +219,9 @@ def get_eap_manifest_version(manifest_dict):
                     if classification:
                         versions.add(classification)
         else:
-            logger.warning('Expected a dictionary of strings for %s',
-                           EAP_HOME_MODULES_VERSION)
+            logger.warning('Expected a dictionary of strings for %s, '
+                           'but found %s.  Invalid value: %s',
+                           EAP_HOME_MODULES_VERSION, type(manifest), manifest)
```

Comment:
Not related to this PR but we might consider going through this file and improving the logging in the rest of the functions. (There are a lot of log statements that don't say what was found, only what was expected)

Reply:
Yeah.  Agree. :(